frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { ListGroup } from 'react-bootstrap';
+import { IParcel } from 'actions/parcelsActions';
+import { Label } from 'components/common/Label';
+import './InfoSlideOut.scss';
+import { formatMoney } from 'utils/numberFormatUtils';
+
+interface IParcelAttributes {
+  /** the selected parcel information */
+  parcelInfo: IParcel | null;
+}
+
+/**
+ * Displays parcel specific information needed on the information slide out
+ * @param parcelInfo the selected parcel data
+ */
+export const ParcelAttributes: React.FC<IParcelAttributes> = ({ parcelInfo }) => {
+  const formatAssessed = parcelInfo?.assessed
+    ? formatMoney(parcelInfo?.assessed)
+    : formatMoney(parcelInfo?.evaluations[0].value);
+  return (
+    <>
+      <ListGroup>
+        <Label className="header">Parcel attributes</Label>
+        <ListGroup.Item>
+          <Label>Lot size:</Label>
+          {parcelInfo?.landArea + ' hectares'}
+        </ListGroup.Item>
+      </ListGroup>
+      <ListGroup>
+        <Label className="header">Legal description</Label>
+        <ListGroup.Item>{parcelInfo?.landLegalDescription}</ListGroup.Item>
+      </ListGroup>
+      <ListGroup>
+        <Label className="header">Valuation</Label>
+        <ListGroup.Item>
+          <Label>Assessed value:</Label>
+          {formatAssessed}
+        </ListGroup.Item>
+      </ListGroup>
+      {parcelInfo?.buildings && (
+        <ListGroup>
+          <Label className="header">Associated Buildings</Label>
+          <ListGroup.Item>
+            <Label>{parcelInfo.buildings[0].name}</Label>
+          </ListGroup.Item>
+          {parcelInfo.buildings[1] && (
```

Comment:
You don't want to manually iterating through a list like this, this is unsafe.
if you want to limit the maximum, create a new array of just the top four elements and then do a .map of that array.

Reply:
checking if parcelInfo.buildings[1] exists doesn't work, because if it doesn't exist javascript will throw an error to check if an array element exists use the .length property or use getIn(someArray, someIndex)